test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -0,0 +1,942 @@
+		"autofail":    "false",
+		"conditional": "true",
+		"check":       "prefix-",
+	})
+	if _, err := client.CoreV1().ConfigMaps("default").Create(context.TODO(), condpassParams, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+	condpassBinding := withBindingExistsLabels([]string{"condpass-binding-label"}, policy, makeBinding("condpass-binding", "test-policy", "condpass-params"))
+	if _, err := client.AdmissionregistrationV1alpha1().ValidatingAdmissionPolicyBindings().Create(context.TODO(), condpassBinding, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+
+	// allow time for reconciliation
+	time.Sleep(time.Second)
+
+	autofailingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "autofailing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"autofail-binding-label": "true",
+			},
+		},
+	}
+	_, err = client.CoreV1().Secrets("default").Create(context.TODO(), autofailingSecret, metav1.CreateOptions{})
+	if err == nil {
+		t.Fatal("expected secret creation to fail due to autofail-binding")
+	}
+	checkForFailedRule(t, err)
+	checkFailureReason(t, err, metav1.StatusReasonInvalid)
+
+	autopassingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "autopassing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"autopass-binding-label": "true",
+			},
+		},
+	}
+	if _, err := client.CoreV1().Secrets("default").Create(context.TODO(), autopassingSecret, metav1.CreateOptions{}); err != nil {
+		t.Fatalf("expected secret creation to succeed, got: %s", err)
+	}
+
+	condpassingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "prefix-condpassing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"condpass-binding-label": "true",
+			},
+		},
+	}
+	if _, err := client.CoreV1().Secrets("default").Create(context.TODO(), condpassingSecret, metav1.CreateOptions{}); err != nil {
+		t.Fatalf("expected secret creation to succeed, got: %s", err)
+	}
+
+	condfailingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "condfailing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"condpass-binding-label": "true",
+			},
+		},
+	}
+	_, err = client.CoreV1().Secrets("default").Create(context.TODO(), condfailingSecret, metav1.CreateOptions{})
+	if err == nil {
+		t.Fatal("expected secret creation to fail due to autofail-binding")
+	}
+	checkForFailedRule(t, err)
+	checkFailureReason(t, err, metav1.StatusReasonInvalid)
+}
+
+func makePolicy(name string) *admissionregistrationv1alpha1.ValidatingAdmissionPolicy {
+	return &admissionregistrationv1alpha1.ValidatingAdmissionPolicy{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	}
+}
+
+func withParams(params *admissionregistrationv1alpha1.ParamKind, policy *admissionregistrationv1alpha1.ValidatingAdmissionPolicy) *admissionregistrationv1alpha1.ValidatingAdmissionPolicy {
```

Comment:
Use these `with*()` utility functions consistently across this tests suite?

Reply:
Note that a unused `with` function is what is causing `pull-kubernetes-verify` to fail.